### PR TITLE
duplicati: livecheck only stable and beta versions

### PR DESCRIPTION
**Pull request by freayd**
Livecheck edits:
- use a git strategy
- only look for stable and beta versions as already discussed with @reitermarkus in #97381

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

### CODE REVIEW

**Comment by SMillerDev** [comment]
I don't even think it should be checking for beta versions.

**Comment by freayd** [comment]
Why not?

File: Casks/duplicati.rb [hunk 1]
```diff
@@ -9,10 +9,12 @@
...
 
   livecheck do
-    url "https://github.com/duplicati/duplicati/releases/latest"
-    strategy :page_match do |page|
-      match = page.match(%r{href=.*?/duplicati-(\d+(?:\.\d+)*)_([^/]*?)_(\d+(?:-\d+)*)\.dmg}i)
-      "#{match[1]},#{match[2]}:#{match[3]}"
+    url :url
+    strategy :git do |tags|
+      tags.map do |tag|
+        match = tag.match(/^v(\d+(?:\.\d+)*)-(?:\d+(?:\.\d+)*)_(stable|beta)_(\d+(?:-\d+)*)$/i)
```
Comment (miccal) [inline Casks/duplicati.rb:15]: I do not see any `stable` releases -- only `beta`, `experimental` and `canary`?

Comment (freayd) [reply Casks/duplicati.rb:15]: Correct. I only added it to ensure an hypothetical stable release wouldn't be ignored.

**Comment by miccal** [comment]
Merging as the `homepage` links to the `beta` version.

**Pull request merged**